sleep.h

```diff
@@ -0,0 +1,8 @@
+#ifndef SLEEP_H
+#define SLEEP_H
+
+#include <nan.h>
+
+void node_usleep(uint32_t usec);
```

Comment:
Changed this to uint32_t because it is one of types supported by Nan, also "unsigned" has undefined size (according to C++ standard, it should be at least 16 bits, it's 32 bits at all modern platforms, but nonetheless it is safer).